### PR TITLE
Add some puppeteer configuration options

### PR DESCRIPTION
**Pull request by simonft**
Includes ability to pass a chrome executable path, a default viewport, and args passed to the chromium executable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,12 @@ Results are stored in `demo-dir` by default
 - `defaultWaitUntil`
   - default: 'networkidle2'
   - [Other options](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagegotourl-options)
+- `executablePath`
```
Comment (dphiffer) [inline README.md:80]: I think this should be `puppeteerExecutablePath`.

File: README.md [hunk 2]
```diff
@@ -77,6 +77,12 @@ Results are stored in `demo-dir` by default
...
+  - Path to Chromium executable.
+  - default: uses bundled puppeteer chromium
+- `extraChromiumArgs`
```
Comment (dphiffer) [inline README.md:83]: I tried passing `["--accept-lang=es-ES"]` to this and it didn't work (it does work independently of the collector), I wonder if there's some other arg I could try?

Comment (simonft) [reply README.md:83]: This works if you are in headful mode, showing it's passing the arg correctly. For some reason chrome doesn't send that header in headless mode.

Comment (dphiffer) [reply README.md:83]: Not our bug!

**Review by dphiffer — APPROVED** [review]

**Pull request merged**